CONTRIBUTING.md

```diff
@@ -89,6 +165,45 @@ We welcome contributions, large or small, including:
 - Example code snippets
 - Sample data
 
-If you would like to contribute new APIs, please check the [CODEOWNERS](https://github.com/iTwin/itwinjs-core/blob/master/.github/CODEOWNERS) file to ensure your contributions align with the package owner's and project goals.
+If you would like to contribute new APIs, create a new [issue](https://github.com/iTwin/itwinjs-core/issues) and explain what these new APIs aim to accomplish. If possible check the [CODEOWNERS](https://github.com/iTwin/itwinjs-core/blob/master/.github/CODEOWNERS) file and tag the owners of the package you plan on introducing the new APIs into.
 
 Thank you for taking the time to contribute to open source and making great projects like iTwin.js possible!
+
+## Frequently Asked Questions
+
+### Rush commands take too long, is there a way to speed up my contribution workflow?
+
+If your source code change only impacts the subdirectory you are working on, you can use a package manager to run local commands, without affecting other packages.
```

Comment:
i do not recommend using any other package manager. if you are working in a subdirectory in this monorepo, they should use rushx. others might work, but should not be recommended

Reply:
Okay, I've explicitly mentioned using 'rushx' to run local commands instead